### DISCUSSION TITLE
Display Data from Table

### DISCUSSION THREAD

**Discussion by ndyaanzr** [eclipse-birt/birt#1784] [Q&A]
Hello,

Please help me with this. Let say I have a table that display data like this.

![g1](https://github.com/eclipse-birt/birt/assets/60999025/d9378b78-78b8-42ed-8de3-97b44a96d002)

When I use Suppress Duplicates on Specialization column, the table will display as below.

![g2](https://github.com/eclipse-birt/birt/assets/60999025/9623a55a-9dbf-41de-8bd4-44e370273f07)
 
If the data for the Specialization column is all the same, is there any way for me to display the data as below?

 Grade: G4 or G5 or G6 or G7, Specialization: E04 & E11.

**Comment by hvbtup** [comment]
You can use a group, use row["Specialization"] as the expression, and use the CONCAT aggregate function.
